src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import copy
+import pytest
+import platform
+from PIL import Image
+
+import torch
+import torch.nn.functional as f
+import torchvision.transforms as transforms
+
+from openvino.runtime import Core, Tensor
+from openvino.tools.mo import convert_model
+
+from openvino.preprocess.torchvision import PreprocessConverter
```

Comment:
Can we make the namespace shorter? `torchvision` looks excessive to me.  Just check out `openvino.preprocess.torchvision.PreprocessConverter.from_torchvision`. I would expect something like: `openvino.preprocess.PreprocessConverter.from_torchvision`

Reply:
Agreed offline to do in a separate PR.